astro/src/content/docs/release-notes/index.mdx

```diff
@@ -38,6 +38,21 @@ import IssueResolvedVia from 'src/components/docs/release-notes/ReleaseNoteIssue
 
 Looking for release notes older than 1.23.0? Look in the [release notes archive](/docs/release-notes/archive). Looking to be [notified of new releases?](/docs/operate/roadmap/releases#release-notifications) <span class="not-prose no-underline"><a class="ml-2" href="/docs/releases.xml"><i class="fas fa-xs fa-rss text-orange-700 text-2xl" width="50px" /></a></span>
 
+<ReleaseNoteHeading version='1.53.3' releaseDate='October 22, 2024'/>
+
+### Fixed
+<Issue issue="2764">
+The CSRF token for federated login added in version `1.47.0` is not applied when all identity providers configured for an application use managed domains, and the OAuth authorize request for the application includes an `idp_hint` parameter.
+</Issue>
+
+<Issue issue="2893,2472">
+FusionAuth is not honoring the `rememberDevice` setting set by a user when performing a federated login. This setting tells FusionAuth whether it should establish an SSO session for the user. FusionAuth will look for this value in the following order: 
```

Comment:
I do think we need to mention the parameter name, but maybe not lead with it? We should help the person reading this release note know if they need to care about this bug fix / improvement. 

Suggestion:


> When using the hosted login pages, the end user is generally shown a checkbox named `Keep me signed in` which indicates the user wishes to create an SSO session during this login request. 
> 
> When you use a 3rd party logins via SAML v2, OpenID Connect, and other 3rd party identity with an `idp_hint` or `login_hint` that may cause the user to bypass the login page and be directed to the 3rd party identity provider, in this scenario the user is not shown this page so they are not given a choice. 
> 
> This behavior has been improved in order to provide additional control on how the SSO session should be created. 
> 
> FusionAuth will now use the following order of operation in this non-interactive workflow to decide if the SSO session should be created. 
> 1. The user's previous selection if available. This past choice will be stored in an HTTP only cookie.
> 2. The optionally supplied `rememberDevice` query parameter. 
> 
> In the event that the user has never seen the login page, the value of the `rememberDevice` query parameter be the deciding factor. A value of `true` will indicate the SSO session should be created, and a value of `false` will indicate the SSO session should not be created. If this parameter is omitted, the default behavior will be to create the SSO session. 

You'll want to confirm this is correct, but I think this is helpful for the user to understand the workflow. I omitted your step 2 because that assumes the user is on the login page, and I don't know that we need to spell out that what the user requests in a normal case behaves as everyone would expect already. 

This copy doesn't explicitly say we are solving a bug - up to you if you want to spell that out. My copy is is more to say, we are now giving the caller more control, which is accurate.

Reply:
Updated